### DISCUSSION TITLE
It is Impossible to see whether module is active in PS 8.1

### DISCUSSION THREAD

**Discussion by musicpanda** [PrestaShop/PrestaShop#33462] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

It is impossible to see in PS 8.1 on the Module Manager page whether a module is active or not when it has settings. It will always show "configure".

In the attached image the first module is active and the second not. There is no way to notice that other than looking under the Configure button.
![modules](https://github.com/PrestaShop/PrestaShop/assets/13930107/cf7d78ac-7163-4a03-966e-ae71a81e4202)

### Expected behavior

There should be some clear visual clue whether a module is active.

### Steps to reproduce

Open the Modules page...

### PrestaShop version(s) where the bug happened

8.1

### PHP version(s) where the bug happened

7.4

**Comment by Hlavtox** [comment]
Hi @musicpanda, this is expected behavior, we want the Configure button to be the main one, because that's the action you do the most when you go to module manager.

However, I like the idea, I made a PR to 8.1.2. ;-) https://github.com/PrestaShop/PrestaShop/pull/33452

![Snímek obrazovky 2023-07-27 142555](https://github.com/PrestaShop/PrestaShop/assets/6097524/d4ed1441-e5ab-4e9d-9ba3-ebd5c85398d5)

**Comment by florine2623** [comment]
ping @PrestaShop/product-team

**Comment by Hlavtox** [comment]
@florine2623 Hi, I just replied, no need to ping product team again, we will resolve it.

**Comment by hibatallahAouadni** [comment]
@Hlavtox you need to migrate this issue to discussion tab cause it's new feature

**Reply by Hlavtox** [reply]
@hibatallahAouadni Done ✅

**Comment by ShaiMagal** [comment]
This is definitely needed.

And like @Caleydon wrote in issue #37044:
"_A visual indication of the module's state is definitely needed. Perhaps uninstalled modules should have a different background color for the entire row. Personally, I use a small modification that displays a red right border for Disabled and Uninstalled modules._"

I think, it can be usefull as well, but most important part is enable/disable state of module. Because it will result in human factor errors.
